helm/teleport-operator/templates/deployment.yaml

```diff
@@ -33,9 +33,12 @@ spec:
         env:
         - name: TELEPORT_TLS_ROUTING_CONN_UPGRADE
           value: "{{ .Values.teleport.proxyAddr }}=yes"
+        - name: USE_COMBINED_TOKENS
+          value: {{ .Values.teleportOperator.useCombinedTokens | quote }}
```

Comment:
This can go away now, right?

Reply:
yeah, removing it, I thought I already had it removed